src/templates/_pages/page_layout.twig

```diff
@@ -0,0 +1,125 @@
+        author: 'Carla Hall',
+        type: 'cooking recipes'
+    }
+] %}
+
+{% set blogsData = [
+    {
+        title: 'Where is the ‘self’ in ‘self-improvement’?',
+        imagePath: 'https://cdn-images-1.medium.com/max/2000/0*kR8CdC0KpGUUoIQV',
+        description: 'Today, many of us think about self-improvement in the same manner we think about home-improvement or car-improvement or culinary-improvement',
+        link: 'https://medium.com/the-polymath-project/where-is-the-self-in-self-improvement-55543fc9cc92',
+        author: 'Charles Chu',
+        type: 'lecture'
+    },
+    {
+        title: 'How I made my own YouTube Downloader using JavaScript and Node.js',
+        imagePath: 'https://cdn-images-1.medium.com/max/2000/0*3XXUJnMnFebws0ia.png',
+        description: 'Downloading videos from YouTube is against the YouTube Policy. The only videos that your allowed to download is your own which you can already do using YouTube Studio.',
+        link: 'https://blog.usejournal.com/how-i-made-my-own-youtube-downloader-using-javascript-and-node-js-160b172f6e10',
+        author: 'Moorad Altamimi',
+        type: 'tech articles'
+    },
+    {
+        title: 'The Dangerous Assumption You Could Be Making About Your Life',
+        imagePath: 'https://cdn-images-1.medium.com/max/1600/0*0MbLOB87GpHBkxQp',
+        description: 'The Dangerous Assumption You Could Be Making About Your Life',
+        link: 'https://medium.com/swlh/you-think-you-know-your-destiny-but-youre-wrong-1337806040c4',
+        author: 'Tom Kuegler',
+        type: 'self improvement'
+    },
+    {
+        title: 'The Mythbuster’s Guide to Saving Money on Energy Bills',
+        imagePath: 'https://www.miss-thrifty.co.uk/wp-content/uploads/2018/09/38154357964_63da2f548e_z-550x377.jpg',
+        description: 'If you have been here a long time, you might remember when I went on ITV Tonight to dispense a masterclass in saving money on energy bills.',
+        link: 'https://www.miss-thrifty.co.uk/the-mythbusters-guide-to-saving-money-on-energy-bills/',
+        author: 'Miss Thrifty',
+        type: 'case study'
+    },
+    {
+        title: 'Carla Hall’s Southern Spoonbread',
+        imagePath: 'https://cupofjo.com/wp-content/uploads/2018/12/Spoonbread-Dressing-copy.jpg',
+        description: 'What are your family’s signature recipes? Celebrity chef Carla Hall swears by a classic spoonbread dressing (or stuffing). ',
+        link: 'https://cupofjo.com/2018/12/carla-hall-spoonbread/',
+        author: 'Carla Hall',
+        type: 'cooking recipes'
+    }
+] %}
+
+{# NAV BAR #}
+<nav>
+    {% block nav %}
+        <a href="index.html">Home</a>
+        <a href="templates/blogs.html">Blogs</a>
+        <a href="templates/article.html">Article</a>
+    {% endblock %}
+</nav>
+
+
+{# END NAV BAR#}
+<div id="content">
+    {% block content %}
+
+    {% endblock %}
+</div>
+</body>
+
+{% block scripts %}
+    <script src="js/index.js"></script>
```

Comment:
please take the script away from `block scripts`
they are default and should be applied everywhere
the task was to support any eventually new scripts
for that, please add a new block below this line - `{% block additional_scripts %}{% endblock %}`